### ISSUE TITLE
NERDTreeFind on a new file in a new directory fails with E718: Funcref required

### ISSUE THREAD

**Issue by mgedmin** [scrooloose/nerdtree#667]
Steps to reproduce:
- open some file
- open nerdtree (e.g. with `:NERDTreeFind`)
- open a new file with `:e new-directory/new-file.txt`
- create the directory with `:!mkdir -p %:h`
- create the file with `:w`
- try to locate the file with `:NERDTreeFind`

What I expect:
- nerdtree refreshes the parent directories and locates the file

What I get:

```
Error detected while processing function <SNR>107_findAndRevealPath[42]..236[23]..236:
line   23:                                                                                                                                      
E716: Key not present in Dictionary: reveal(a:path, opts)
E15: Invalid expression: n.reveal(a:path, opts)
Error detected while processing function <SNR>107_findAndRevealPath:
line   44:
E718: Funcref required
```

**Comment by lifecrisis** [comment]
This bug has been confirmed.

**Comment by lifecrisis** [comment]
This was fixed as part of PR #785.  Things should work if you update the NERDTree.  Re-open if that's not the case.
